kubernetes-dev-environment.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     spec:
       restartPolicy: Never
       containers:
-        - name: dev-environment
-          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp:dev-environment
+        - name: allennlp-dev-env
+          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp/allennlp-gpu:latest
```

Comment:
I tried to mirror the workflow of `run_on_aws.sh` for now.  I'm not really sure where this should go.  I think this script will disappear once we move to beaker?  That probably won't happen until post-launch however.